docker/pypi/wmagent-base/Dockerfile

```diff
@@ -13,3 +14,11 @@ COPY --from=oracle /usr/lib/oracle /usr/lib/oracle
 ENV LD_LIBRARY_PATH=/usr/lib/oracle
 ENV PATH=$PATH:/usr/lib/oracle
 ENV PKG_CONFIG_PATH=/usr/lib/oracle
+
+# overwrite /etc/pam.d/login such that we can access accounts at CERN
+RUN printf "\n\
```

Comment:
Can we use the same pattern with a heredoc everywhere we need to edit documents at the image? 

like: 
```
RUN <<EOF cat >> /etc/pam.d/login
auth required pam_sss.so
account required pam_sss.so
password required pam_sss.so
session required pam_sss.so
EOF
```
It avoids syntactic complications and workarounds  with escape characters etc.